server/controllers/operations.coffee

```diff
@@ -42,50 +42,73 @@ module.exports.delete = (req, res) ->
 
 module.exports.query = (req, res) ->
 
-    paramAccounts = req.body.accounts or [-1]
-
-    BankOperation.allFromBankAccounts paramAccounts, (err, operations) ->
-        if err?
-            res.send 500, error: 'Server error occurred while retrieving data'
+        paramAccounts = req.body.accounts or [-1]
```

Comment:
I suspect you haven't changed anything to .query but the indentation, can you put it back to what it was? (and squash your commits, please)